views/profile.pug

```diff
@@ -19,9 +19,10 @@ block content
        div.divider
        //- formulier voor profile editten: dog preferences
        label(for="dogAge") Max dog age
-       input(type="number" id="dogAge" name="dogAge" value=user.maxAge)
+       input(type="number" id="dogAge" name="maxAge" value=user.maxAge)
        label(for="breed") preferred breeds
        input(type="search" id="breed" name="breed" value="insert breed")
```

Comment:
Misschien kan je hier beter voor value placeholder gebruiken, zodat de value niet "insert breed" is maar user.breed

Reply:
Oooh ja die komt er hoor, ik had het nu alleen als insert breed gezet voor mezelf als reminder omdat ik daar de rassen nog in moet zetten ;)

Reply:
Topper sam